lms/djangoapps/courseware/testutils.py

```diff
@@ -90,27 +118,60 @@ def setup_user(self, admin=False, enroll=False, login=False):
         if login:
             self.login()
 
+    def set_block_to_be_requested(self, block_name):
+        """
+        Sets the block currently being tested.  This allows the positive and
+        negative tests to refer to the same block.
+
+        Arguments:
+            block_name: The name of the block (e.g. 'html_block'), that should
+                match both the attribute for the block as well as the key into
+                BLOCK_SPECIFIC_ELEMENTS.  Use None to clear values.
+
+        Side Effects:
+            Sets block_to_be_requested to the block under test.
+            Sets block_specific_chrome_html_elements to an array of elements that
+               should appear in the courseware chrome specifically for this block.
+        """
+        if block_name is None:
```

Comment:
I agree.  I think this code will land in the course setup code which will ensure that it stays in sync.  I'll update and we'll see.